Assets/Scripts/Algorithms/Patrolling/Components/CollisionRecoverySenseNeighboringRobotsComponent.cs

```diff
@@ -0,0 +1,125 @@
+// Copyright 2025 MAEPS
+// 
+// This file is part of MAEPS
+// 
+// MAEPS is free software: you can redistribute it and/or modify it under
+// the terms of the GNU General Public License as published by the
+// Free Software Foundation, either version 3 of the License, or (at your option)
+// any later version.
+// 
+// MAEPS is distributed in the hope that it will be useful, but WITHOUT
+// ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+// or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General
+// Public License for more details.
+// 
+// You should have received a copy of the GNU General Public License along
+// with MAEPS. If not, see http://www.gnu.org/licenses/.
+//
+// Contributors 2025: 
+// Casper Nyvang Sørensen,
+// Christian Ziegler Sejersen,
+// Jakob Meyer Olsen
+// Mads Beyer Mogensen
+using System.Collections.Generic;
+using System.Text;
+
+using Maes.Map.PathFinding;
+using Maes.Robot;
+using Maes.Robot.Tasks;
+
+namespace Maes.Algorithms.Patrolling.Components
+{
+    internal class CollisionMessage
```

Comment:
Put this inside CollisionRecoverySenseNeighboringRobotsComponent and make it private sealed

Reply:
I forgot to remove it, it was old code